Core/src/org/sleuthkit/autopsy/datamodel/DeletedContent.java

```diff
@@ -386,25 +391,19 @@ public void update(Observable o, Object arg) {
             }
 
             @Override
-            protected void addNotify() {
+            protected void onAdd() {
```

Comment:
Do we need to use the Template Method design pattern here? The superclass addNotify method seems to do nothing other than call the onAdd method...or is there more coming in a later PR?